src/docs/README.md

```diff
@@ -0,0 +1,41 @@
+# Website
+
+This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+
+### Installation
+
+```
+$ yarn
```

Comment:
I don't think you wrote this, but let's avoid having people use `yarn`, and prefer `pnpm`

Reply:
Updated this to use pnpm. And yes, I didn't write this. These were the default docs for docusaurus.